Makefile

```diff
@@ -12,7 +12,7 @@ BUILD_DIR ?= $(CURDIR)/out
 
 GO_OS ?= $(shell go env GOOS)
 GO_ARCH ?= $(shell go env GOARCH)
-GO_PACKAGES := $(shell go list ./... | grep -vE "vendor|tools")
+GO_PACKAGES := $(shell go list ./... | grep -vE "vendor|tools|sweep")
```

Comment:
Adding the sweep package to the skipped go packages as we can skip reporting on test coverage for that as it's part of the testing infra.